docs/source/man.rst

```diff
@@ -655,15 +659,49 @@ Insights
 
 Use the ``qpc insights`` command to interact with Red Hat Insights and its services.
 
-Uploading to Insights
-~~~~~~~~~~~~~~~~~~~~~
-The ``qpc insights upload`` command can be used to upload an insights report to Red Hat Insights and its services. You can upload a report by using the associated report identifier or scan job identifier for the scan that is used to generate the report.
+Configuring Insights
+~~~~~~~~~~~~~~~~~~~~
+
+To configure the connection to Insights server, the host address and port can be optionally supplied. They have default values assigned if not provided.
+
+**qpc insights config --host=** *host* **[--port=** *port* **]** **[--use-http=** *(True | False)* **]**
+
+``--host=host``
+
+  Optional. Sets the host address for insights, the default address is ``console.redhat.com``.
+
+``--port=port``
+
+  Optional. Sets the port to use to connect to insights. The default is ``443``.
+
+``--use-http=(True | False)``
```

Comment:
Is this correct ?  In the CLI I thought it was a boolean in that, if not specified it's https, if specified it's http.  so just ``--use-http`` to True or False specified

Reply:
you are correct! I will correct this.